internal/k8s/entity.go

```diff
@@ -83,3 +83,14 @@ func ToLoadBalancer(entity K8sEntity) (LoadBalancer, bool) {
 
 	return result, true
 }
+
+// Get Deployment name
+func GetDeploymentName(entity K8sEntity) string {
```

Comment:
It didn't seem like the deployment name was available elsewhere so I copied and modified the `ToLoadBalancer` function to do my bidding. Happy to do something else if I'm missing somewhere else where this data already exists.

Reply:
ya, this is a bit confusing. in all our yaml files, the service name and the deployment name are the same, but there's no guarantee the user is doing this

let me write a quick function to get the Name of an entity, which is tricky, and i think will help with this

the 'deploy' part of 'kubectl get deploy' should be in `entity.GroupVersionKind.Kind`